autoload/conflict_marker/detect.vim

```diff
@@ -5,19 +5,14 @@ function! conflict_marker#detect#markers() abort
     if !g:conflict_marker_enable_detect
         return
     endif
-    let pos_save = getpos('.')
-    try
-        keepjumps call cursor(1, 1)
-        for marker in [g:conflict_marker_begin, g:conflict_marker_separator, g:conflict_marker_end]
-            if search(marker, 'cW') == 0
-                return 0
-            endif
-        endfor
 
-        return 1
-    finally
-        call setpos('.', pos_save)
-    endtry
+    for marker in [g:conflict_marker_begin, g:conflict_marker_separator, g:conflict_marker_end]
+        if search(marker, 'cnw') == 0
+            return 0
+        endif
+    endfor
+
+    return 1
```

Comment:
This change does not check the order of markers. For example, the following text is detected as it contains conflict marker.

```
>>>>>>>>>>
<<<<<<<<<<
==========
```

Reply:
makes sense. Dropped that commit.